### PR TITLE
*: add codegen for Gopkg.toml

### PR DESCRIPTION
**Pull request by fanminshi**
cc/ @hasbro17

### CODE REVIEW

**Comment by fanminshi** [comment]
Test:
```sh
$ operator-sdk new test-operator
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/cmd/manager/main.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/apis/apis.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/controller/controller.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/build/Dockerfile
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/pkg/build/build.sh
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/deploy/role.yaml
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/deploy/role_binding.yaml
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/deploy/operator.yaml
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/scripts/codegen/boilerplate.go.txt
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/version/version.go
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/.gitignore
Create /Users/fanminshi/work/src/github.com/operator-framework/test-operator/Gopkg.toml
Run dep ensure ...
Root project is "github.com/operator-framework/test-operator"
...
Run dep ensure done
Run git init ...
Initialized empty Git repository in /Users/fanminshi/work/src/github.com/operator-framework/test-operator/.git/
Run git init done
$ cat test-operator/Gopkg.toml
# Force dep to vendor the code generators, which aren't imported just used at dev time.
required = [
  "k8s.io/code-generator/cmd/defaulter-gen",
  "k8s.io/code-generator/cmd/deepcopy-gen",
  "k8s.io/code-generator/cmd/conversion-gen",
  "k8s.io/code-generator/cmd/client-gen",
  "k8s.io/code-generator/cmd/lister-gen",
  "k8s.io/code-generator/cmd/informer-gen",
  "k8s.io/code-generator/cmd/openapi-gen",
  "k8s.io/gengo/args",
]

[[override]]
  name = "k8s.io/code-generator"
  # revision for tag "kubernetes-1.11.2"
  revision = "6702109cc68eb6fe6350b83e14407c8d7309fd1a"

[[override]]
  name = "k8s.io/api"
  # revision for tag "kubernetes-1.11.2"
  revision = "2d6f90ab1293a1fb871cf149423ebb72aa7423aa"

[[override]]
  name = "k8s.io/apiextensions-apiserver"
  # revision for tag "kubernetes-1.11.2"
  revision = "408db4a50408e2149acbd657bceb2480c13cb0a4"

[[override]]
  name = "k8s.io/apimachinery"
  # revision for tag "kubernetes-1.11.2"
  revision = "103fd098999dc9c0c88536f5c9ad2e5da39373ae"

[[override]]
  name = "k8s.io/client-go"
  # revision for tag "kubernetes-1.11.2"
  revision = "1f13a808da65775f22cbf47862c4e5898d8f4ca1"

[[override]]
  name = "sigs.k8s.io/controller-runtime"
  version = "v0.1.3"

[[constraint]]
  name = "github.com/operator-framework/operator-sdk"
  # The version rule is used for a specific release and the master branch for in between releases.
  branch = "master"
  # version = "=v0.0.6"

[prune]
  go-tests = true
  non-go = true
  unused-packages = true

  [[prune.project]]
    name = "k8s.io/code-generator"
    non-go = false
    unused-packages = false
```

File: pkg/scaffold/gopkgtoml.go [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+  revision = "103fd098999dc9c0c88536f5c9ad2e5da39373ae"
+
+[[override]]
+  name = "k8s.io/client-go"
+  # revision for tag "kubernetes-1.11.2"
+  revision = "1f13a808da65775f22cbf47862c4e5898d8f4ca1"
+
+[[override]]
+  name = "sigs.k8s.io/controller-runtime"
+  version = "v0.1.3"
```
Comment (hasbro17) [inline pkg/scaffold/gopkgtoml.go]: Change this to `revision = "53fc44b56078cd095b11bd44cfa0288ee4cf718f"`
That includes the merge for my namespace support PR.
https://github.com/kubernetes-sigs/controller-runtime/commit/53fc44b56078cd095b11bd44cfa0288ee4cf718f

Comment (fanminshi) [reply pkg/scaffold/gopkgtoml.go]: cool

Comment (fanminshi) [reply pkg/scaffold/gopkgtoml.go]: fixed!

**Pull request merged**